satpy/writers/cf_writer.py

```diff
@@ -822,6 +822,17 @@ def save_datasets(self, datasets, filename=None, groups=None, header_attrs=None,
                 header_attrs = flatten_dict(header_attrs)
             root.attrs = encode_attrs_nc(header_attrs)
 
+        # Add header attributes needed for satpy_cf_reader
+        root.attrs["instrument"] = datasets[0].attrs["sensor"]
+
+        _history_create = 'Created by pytroll/satpy on {}'.format(datetime.utcnow())
+        if 'history' in root.attrs:
+            if isinstance(root.attrs['history'], list):
+                root.attrs['history'] = ''.join(root.attrs['history'])
+            root.attrs['history'] += '\n' + _history_create
+        else:
+            root.attrs['history'] = _history_create
+
```

Comment:
I think this is fixed in `main`, right?

https://github.com/pytroll/satpy/blob/acd074530f1eeaa860c1d18de22ecbfd75be8166/satpy/writers/cf_writer.py#L581-L588

Reply:
Yep true. I must have missed this while resolving the merge conflicts.